tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial6.ipynb

```diff
@@ -0,0 +1,666 @@
cfJtjuRwaT9m4i35b6ZT2X3d1uq91YRUDSx09xXmPZaa+h6te7rHG97HwV2bD4qXRZXG8sfQfb71rWf/AIm2PiQsM0dGlLTBYMlRoo5jwL2mwPG1lmQyE71minBabjhxVOC8qXEMN6XaOvRQbbp48xvUST1Ab/f8ewq5doYQ1rnHcLn+nbxHrVNY1TEuzOsc549lsres3J8cp61dSWpmuPdNXQUeUwNI1s+V1+Ga2QHh5oII63DqWZgFH5zj50j3ey27neGYtA4hzXjcs6lpsz3luriWtZfjrqOuxsSSd29ZdG20mhuCRGzj0GEl79PxPJcT+YhaJSMaRqceBy9RcSD3jLb36Fam3SjPWcvgS0gacekfZK2G0LrZb8Gvc46/fDXHfroHBdVJEejfex8B7r6Hv1dw6lWtibWphVrAGSt7vc8j4W9S0dM33ZdO9uv6rd40LOmHaf8Arl09wWqhZax4Etv4Zr+/9VOOx4znHpm42aD6h/Vo9la7EG9LUdEm3cRe1+8D3FbRjdT2xuHi3PY+s+9YMZuO8WPZfS/vCsg8Mrmso0NXDrp3+vX3ajwXCjqXMcHMcWuabhzdCD2Lvkfa19NPZN/f/TuXS9l9Rv4ramYWizNjdqhPaOWzZraHcJbdXU7rbx3jiBKVQ8MhBBBIIIII0II1BBGoIKtrYvHfrEXStz0dg8fiHCQd9tRwIPYvSJvkRF6AirHyg1Ho6f2ZPmp5Qaj0dP7MnzUBZyKsfKDUejp/Zk+anlBqPR0/syfNQFnIqx8oNR6On9mT5qeUGo9HT+zJ81AWcirHyg1Ho6f2ZPmp5Qaj0dP7MnzUBZyKsfKDUejp/Zk+anlBqPR0/syfNQFnIqx8oNR6On9mT5qeUGo9HT+zJ81AWcirHyg1Ho6f2ZPmp5Qaj0dP7MnzUBZyKsfKDUejp/Zk+anlBqPR0/syfNQFnIqx8oNR6On9mT5qeUGo9HT+zJ81AWcirHyg1Ho6f2ZPmp5Qaj0dP7MnzUBZyKsfKDUejp/Zk+anlBqPR0/syfNQFnIqx8oNR6On9mT5qeUGo9HT+zJ81AWcirHyg1Ho6f2ZPmp5Qaj0dP7MnzUBYmL/ANzJ/wDbf/0lQGqdlLSAOHZc3HHs/fBYtRt5O5paY4LOBabNkvYix/3q1Zx9+nQiNjfpNLh6nOI9S9TPGjeY83LOx+oF23O/S1iLcTYld+It+zOXUju7r3vbeQfBaKp2okd50cGttzXDQfdFn2AvrcWPasaTG3m9ww3tc2dwv+btKZGDf7B0BfXwZgchlANuDBrI7dbSME99upesxIXQ5dHFuWN3VmY3JI3fv50keA3rx1gG1stPK2SOOEuYHNAeJCOmC0k2kBvZx4qYQ8uNcA4c1RnOXuLjHNfNI4yOd/tFr5nE7liuaM6j0NltWjT3Lf2lrOnEL6in5zq3km/jYepRnB4s1bl1AfGHX6vtA646+jmFuO7iqwrOVKqe4F0dNcRNiFmy2DWgi+s283N+HYuul5TKlkgeI6bMG5NWSWtp1TX4DisqsqnY1u8p9yydqiCbjc7niB/G1tzqOsHxzd61lNHmpZGjeCGcOFxc+Nj4qv6vb+d4s6On3EaNkGh/4vBcY9u5w1zQyCz3FxOWS9zl3fa7uj7z4WRtZorldQZsNq47sG67mROPi1tz7TR/N1LQ07Ljq0v6tPiPiuvEtopJXXcyMdBrLNDwLNJcDbPv1I7iViMxNwFsrLa/i46fiWmNOSWCh1otm9c28d7Wu4WvwBBDh4EALDg0DgfynwFz8BbvWG3GX5Q3LHYbtHX6/wAfXc+J7LdRxN1z0Wa/xf8Al3+sr32bHtok32cms4a79Qf3w3q19lqnI+N5vZ32b91jfVh8Dp/i7F56pNoJGea2PTrDju/x9/rUgpeUypYLCOmIPAtl+cs8rab2NNO7gtz1vQsCkMUYLfVdeRKPl2r2AARURt+KOY/CoC2MX0jsSH+4w7xiqP0qlD8LMud9S7/I9AbW0GbMODWX8Tf4jN6lUm0eG2cCR0WuFmjS9sxF+oagX3/FRKr+kDiDzrBQakE2jn1yiwBvU7tSVoqnlZqnG5hpPYl468Z+tFa1EyE7ylJcyeQREXva7nhotplb5ugO65JHXYO4lc4orzn8MbDoNwv+uo77BVm/lHqSbmOm1Id5snAgj/e9gSLlHqQHWjp7vJJJbLcE33fbWG9TdvMze3gSXaY3eepx9QzDT1BvuXdOzLl3j7XUfwAj3f0UDl2vlcblkO++5/zOwLsn20mda8cAsSdBJvP/ABe71J+Hme/iIEix3/aJh1m4HUH2f8HrW0QuztBBt/iyH4g+BWkqdpZHSF5ZFmcGAgB9vs2NjH3+IaL9t11R488X6EevY/rv+Pr+KmqEkRdeJv5Tobb25hr13uPgSsCiNyR3Ed17fAhax+NPN+izU33O7fzdpXVDibmm4azxzf8Akp+xeCPtonbikVrdpcPVlsfELCY+1v34Fd1TXlwsWt8M3/ksfnPyj+b+qvinjUzTab0Oc4G8bjvHUVn7MYoYJ2v+6DZ462OtmHhvHa0LXNl/K3X+L/yXEP8Ayj+b/wAlIiXy1wIuDcEAgjiDqCvqqrD9uJ442sayEtjaGgubITYaC9pBuGngu/yg1Ho6f2ZPmr0EQREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREB//9k=\n",
```

Comment:
Whats going on with this image?

Reply:
sorry, couldn't locate the video in the tutorial. Seems like that is coming out of the video rendering cell